server/lib/dynamoDao.js

```diff
@@ -0,0 +1,54 @@
+const AWS = require('aws-sdk');
+var dynamodb = new AWS.DynamoDB({
+    region: 'us-east-1',
+    endpoint: 'http://localhost:8000',
+});
+
+AWS.config.update({
+    region: 'us-east-1',
+    endpoint: 'http://localhost:8000',
+});
+var docClient = new AWS.DynamoDB.DocumentClient();
+function queryDynamoByChannel(channelId) {
+    const params = {
+        TableName: 'twitch-sessions',
+        Key: {
+            channelId: { S: `channel${channelId}` },
```

Comment:
Why do we need to prepend `channel` to the ID? Is there a reason we need to store the channel id with that prefix? It's baggage for no reason. The name of the field tells us what the purpose is. We don't need to transform the value we get from Twitch just to prepend a constant `"channel"` to it.